src/settings.ts

```diff
@@ -15,6 +9,109 @@ function isMobileBrowser(): boolean {
   return check;
 }
 
+function getWeb3ConnectProviderOptions(network: string) {
+  if (typeof(window) === "undefined") {
+    return null;
+  }
+
+  const BurnerConnectProvider: any = null; // require("@burner-wallet/burner-connect-provider");
```

Comment:
Web3Connect won't work without an imported module, last I checked.  Definitely a good idea to lazy-load them and keep them in separate chunks, but I don't think `null` is going to work.

Reply:
oops, i forgot to uncomment that